src/azure-cli/azure/cli/command_modules/cosmosdb/tests/latest/test_cosmosdb_commands.py

```diff
@@ -1410,10 +1412,77 @@ def test_cosmosdb_managed_service_identity(self, resource_group, key_vault):
         assert cmk_account["defaultIdentity"] == 'SystemAssignedIdentity'
 
+        id_account = self.cmd('az cosmosdb create -n {acc} -g {rg} --assign-identity [system]').get_output_in_json()
+        assert id_account["identity"]['type'] == 'SystemAssigned'
+
+        identity_output = self.cmd('az cosmosdb identity remove -n {acc} -g {rg}').get_output_in_json()
+        assert identity_output["type"] == "None"
+
+        identity_output = self.cmd('az cosmosdb identity assign -n {acc} -g {rg}').get_output_in_json()
+        assert identity_output["type"] == "SystemAssigned"
+
+        # User assigned identity tests
+        user_identity1 = self.cmd('az identity create -n {id1} -g {rg}').get_output_in_jason()
+        user_identity2 = self.cmd('az identity create -n {id2} -g {rg}').get_output_in_jason()
+
+        identity_output = self.cmd('az cosmosdb identity assign -n {acc} -g {rg} --identities {0}', user_identity1["id"]).get_output_in_json()
+        assert identity_output["type"] == "SystemAssigned,UserAssigned"
+        assert identity_output["user_identities"][0][0] == user_identity1["id"]
+        assert len(identity_output["user_identities"]) == 1
+
+        identity_output = self.cmd('az cosmosdb identity assign -n {acc} -g {rg} --identities {0}', user_identity2["id"]).get_output_in_json()
+        assert identity_output["type"] == "SystemAssigned,UserAssigned"
+        assert identity_output["user_identities"][1][0] == user_identity2["id"]
+        assert len(identity_output["user_identities"]) == 2
+
+        identity_output = self.cmd('az cosmosdb identity remove -n {acc} -g {rg}').get_output_in_json()
+        assert identity_output["type"] == "UserAssigned"
+        assert len(identity_output["user_identities"]) == 2
+
+        identity_output = self.cmd('az cosmosdb identity remove {0} -n {acc} -g {rg}', user_identity2["id"]).get_output_in_json()
+        assert identity_output["type"] == "UserAssigned"
+        assert identity_output["user_identities"][0][0] == user_identity1["id"]
+        assert len(identity_output["user_identities"]) == 1
+
+        identity_output = self.cmd('az cosmosdb identity assign -n {acc} -g {rg} --identities {0}', user_identity2["id"]).get_output_in_json()
+        assert identity_output["type"] == "UserAssigned"
+        assert identity_output["user_identities"][1][0] == user_identity2["id"]
+        assert len(identity_output["user_identities"]) == 2
+
+        identity_output = self.cmd('az cosmosdb identity remove -n {acc} -g {rg} --identities {0} {1}', user_identity1["id"], user_identity2["id"]).get_output_in_json()
+        assert identity_output["type"] == "None"
+
+        identity_output = self.cmd('az cosmosdb identity assign -n {acc} -g {rg} --identities {0} {1} [system]', user_identity1["id"], user_identity2["id"]).get_output_in_json()
+        assert identity_output["type"] == "SystemAssigned,UserAssigned"
+        assert len(identity_output["user_identities"]) == 2
+        
```

Comment:
Another Bug I found here is, if there's only 1 UA on the account, and then I run az cosmosdb remove xxx --identities [system] UA1, it'll pass.

The expected behavior should be fail because the SystemAssigned is not associate with the account.